Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/config.py

```diff
@@ -8,16 +8,123 @@
 #
 #
 # -------------------------------------------------------------------------
-# DCCsi\\Tools\\DCC\\Maya\\config.py
+"""! @brief
+Module Documentation:
+    < DCCsi >:: Tools/DCC/Maya/config.py
 
-"""DccScriptingInterface (DCCsi)
-This is the dynamic config (dynaconf) for O3DE Maya DCCsi interface
+This module manages the dynamic config and settings for boostrapping Blender
 """
+# -------------------------------------------------------------------------
+# standard imports
+import sys
+import os
+import re
+import site
+import inspect
+import traceback
+import importlib.util
+from pathlib import Path
+import logging as _logging
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+# global scope
+_MODULENAME = 'Tools.DCC.Maya.config'
+_MODULENAME = 'Tools.DCC.Maya.setup'
+_LOGGER = _logging.getLogger(_MODULENAME)
+_LOGGER.debug('Initializing: {}.'.format({_MODULENAME}))
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+# Maya is frozen
+#_MODULE_PATH = Path(__file__)
+# https://tinyurl.com/y49t3zzn
+# module path when frozen
+_MODULE_PATH = Path(os.path.abspath(inspect.getfile(inspect.currentframe())))
+_LOGGER.debug('_MODULE_PATH: {}'.format(_MODULE_PATH))
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSI_TOOLS_MAYA = Path(_MODULE_PATH.parent)
+_PATH_DCCSI_TOOLS_MAYA = Path(os.getenv('PATH_DCCSI_TOOLS_MAYA', _PATH_DCCSI_TOOLS_MAYA.as_posix()))
+site.addsitedir(_PATH_DCCSI_TOOLS_MAYA.as_posix())
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSI_TOOLS_DCC = Path(_PATH_DCCSI_TOOLS_MAYA.parent)
+_PATH_DCCSI_TOOLS_DCC = Path(os.getenv('PATH_DCCSI_TOOLS_DCC', _PATH_DCCSI_TOOLS_DCC.as_posix()))
+site.addsitedir(_PATH_DCCSI_TOOLS_DCC.as_posix())
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSI_TOOLS = Path(_PATH_DCCSI_TOOLS_DCC.parent)
+_PATH_DCCSI_TOOLS = Path(os.getenv('PATH_DCCSI_TOOLS', _PATH_DCCSI_TOOLS.as_posix()))
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSIG = Path(_PATH_DCCSI_TOOLS.parent)
+_PATH_DCCSIG = Path(os.getenv('PATH_DCCSIG', _PATH_DCCSIG.as_posix()))
+site.addsitedir(_PATH_DCCSIG.as_posix())
+
+# now we have azpy api access
+from azpy.env_bool import env_bool
+from azpy.constants import ENVAR_DCCSI_GDEBUG
+from azpy.constants import ENVAR_DCCSI_DEV_MODE
+from azpy.constants import ENVAR_DCCSI_LOGLEVEL
+from azpy.constants import ENVAR_DCCSI_GDEBUGGER
+from azpy.constants import FRMT_LOG_LONG
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+# _settings.setenv()  # doing this will add the additional DYNACONF_ envars
+def get_dccsi_config(PATH_DCCSIG=_PATH_DCCSIG.as_posix()):
+    """Convenience method to set and retreive settings directly from module."""
```

Comment:
Run spell check- seeing lots of misspelled words (retrieve) .... bootstrap is missing the "t" in quite a few locations

Reply:
nit: I'll do this next PR, this module will get an overhaul anyway